src/System.Security.Permissions/src/System/Xaml/Permissions/XamlLoadPermission.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Security.Permissions;
+
+namespace System.Xaml.Permissions
+{
+    [Serializable]
+    public sealed class XamlLoadPermission : CodeAccessPermission, IUnrestrictedPermission
+    {
+        private IList<XamlAccessLevel> _emptyAccessLevel = new ReadOnlyCollection<XamlAccessLevel>(Array.Empty<XamlAccessLevel>());
+        public XamlLoadPermission(PermissionState state) { }
+        public XamlLoadPermission(XamlAccessLevel allowedAccess) { }
+        public XamlLoadPermission(IEnumerable<XamlAccessLevel> allowedAccess) { }
+#if NETCOREAPP3_0
+        [ComVisible(false)]
+        public override bool Equals(object obj) { return false; }
+        [ComVisible(false)]
+        public override int GetHashCode() { return base.GetHashCode(); }
+#endif 
+        public IList<XamlAccessLevel> AllowedAccess 
+        { 
+            get 
+            {
+                return _emptyAccessLevel;
+            }
+            private set
+            {
+            }
+        }
+        public override IPermission Copy() { return default(IPermission); }
+        public override void FromXml(SecurityElement elem) { }
+        public bool Includes(XamlAccessLevel requestedAccess) { return false;  }
```

Comment:
This makes sense, I went through the permissions again to update the methods that potentially incorrectly restrict access such as `includes` and `IsSubsetOf` in order to return true so that it allows the user code to continue to function.